libnmstate/netinfo.py

```diff
@@ -81,6 +81,9 @@ def capabilities():
     if nm.ovs.has_ovs_capability():
         caps.add(nm.ovs.CAPABILITY)
 
+    if nm.ovs.is_ovs_running():
+        caps.add(nm.ovs.SERVICE)
```

Comment:
I don't think is required. The `nm.ovs.CAPABILITY` is enough, it's user's job to make sure both daemon and plugin installed. Our job is raise proper error guiding them to do so.

Reply:
I am doing this in order to raise different error messages. IMO, we should tell the user what is missing. What do you think? @tyll Thanks!

Reply:
I think `nm.ovs.CAPABILITY` indicate OVS is functional, or you need to rename it to `nm.ovs.NM_PLUGIN_READY`. For proper error message, you can check `is_ovs_running()` before raise exception.

Reply:
I see. Will do, thanks.

Reply:
Changes done, thanks!